agent/bench.go

```diff
@@ -1271,8 +1339,8 @@ func (b *Bench) runKubeBench(bench share.BenchType, script, remediationFolder st
 			config = remediationFolder + "worker"
 		}
 	}
-
-	cmd = exec.Command("sh", script, config)
+	// Read the journal string for K3s to check if some of the command arguments are set correctly.
+	cmd = exec.Command("sh", script, config, journalStr)
```

Comment:
Question:  maybe it's a little like a corner case, but if the host has both k3s and k8s running, what should the result be?

Reply:
the journalStr will only be used in k3s cases, the k8s can ignore such input.

Reply:
or you think it will be better if we separate like the followiing?
```go
	var journalStr string
	var journalErr error
	if b.isK3s {
		journalStr, journalErr = b.readJournal()
		if journalErr != nil {
			log.WithFields(log.Fields{"error": journalErr}).Error("Error while read the journal")
		}
		cmd = exec.Command("sh", script, config, journalStr)
	} else {
	        cmd = exec.Command("sh", script, config)
	}
```

Reply:
It's more like a general question.  I should have put the comment elsewhere.  Sorry for the confusion.

My understanding is that when enforcer detects a new program running and the program meets a certain criteria, it will trigger the kubebench scan.  

So my question is, when both `kube-apiserver` and `k3s-agent` run at the same node, I think k3s will take the precedence.  Is that right?

Reply:
yes, if both kube-apiserver and k3s-agent run at the same node, it will run wih normal k8s not k3s cis benchmark